### PR TITLE
Merge main to feature/rds-phase-2

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes
Merge main into feature branch rds-phase-2
## Changes

* Built and deployed new Docker image for cumuluss/async-operation

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -382,13 +384,59 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+`{"forceRulesMigration": true}`:
+
+```shell
+aws lambda invoke --function-name $PREFIX-data-migration1 \
+  --payload $(echo '{"forceRulesMigration": true}' | base64) $OUTFILE
+```
+
+### Added
+
+- **CUMULUS-2846**
```
Comment (markdboyd) [inline CHANGELOG.md:401]: I think I would expect to see these CHANGELOG lines from `master` here?

https://github.com/nasa/cumulus/blame/master/CHANGELOG.md#L29-L32

Comment (markdboyd) [reply CHANGELOG.md:401]: Nevermind. It seems like the reason why is that these changes are from `master` before 2845 was merged into it. And we shouldn't add the latest from `master` into this PR, we'll just do another PR subsequently.

File: packages/api/tests/endpoints/test-rules.js [hunk 2]
```diff
@@ -1001,6 +1019,662 @@ test('put() does not write to Dynamo/PostgreSQL if writing to Elasticsearch fail
...
+      collection: {
+        name: pgCollection.name,
+        version: pgCollection.version,
+      },
+      provider: pgProvider.name,
+    }
+  );
+
+  t.truthy(originalDynamoRule.rule.arn);
+  t.truthy(originalPgRecord.arn);
```
Comment (markdboyd) [inline packages/api/tests/endpoints/test-rules.js:1053]:
```suggestion
  t.truthy(originalPgRecord.arn);
  t.truthy(originalEsRecord.rule.arn);
```

File: packages/api/tests/endpoints/test-rules.js [hunk 3]
```diff
@@ -1001,6 +1019,662 @@ test('put() does not write to Dynamo/PostgreSQL if writing to Elasticsearch fail
...
+      collection: {
+        name: pgCollection.name,
+        version: pgCollection.version,
+      },
+      provider: pgProvider.name,
+    }
+  );
+
+  t.truthy(originalDynamoRule.rule.arn);
+  t.truthy(originalDynamoRule.rule.logEventArn);
```
Comment (markdboyd) [inline packages/api/tests/endpoints/test-rules.js:1157]:
```suggestion
  t.truthy(originalDynamoRule.rule.arn);
  t.truthy(originalDynamoRule.rule.logEventArn);
  t.truthy(originalEsRecord.rule.arn);
  t.truthy(originalEsRecord.rule.logEventArn);  
```

File: packages/api/tests/endpoints/test-rules.js [hunk 4]
```diff
@@ -1001,6 +1019,662 @@ test('put() does not write to Dynamo/PostgreSQL if writing to Elasticsearch fail
...
+      provider: pgProvider.name,
+    }
+  );
+
+  const expectedMeta = {
+    visibilityTimeout: 10,
+    retries: 3,
+  };
+  t.deepEqual(originalDynamoRule.meta, expectedMeta);
+  t.deepEqual(originalPgRecord.meta, expectedMeta);
```
Comment (markdboyd) [inline packages/api/tests/endpoints/test-rules.js:1286]:
```suggestion
  t.deepEqual(originalPgRecord.meta, expectedMeta);
  t.deepEqual(originalEsRecord.meta, expectedMeta);
```

File: packages/api/tests/endpoints/test-rules.js [hunk 5]
```diff
@@ -1001,6 +1019,662 @@ test('put() does not write to Dynamo/PostgreSQL if writing to Elasticsearch fail
...
+      },
+      collection: {
+        name: pgCollection.name,
+        version: pgCollection.version,
+      },
+      provider: pgProvider.name,
+    }
+  );
+
+  t.truthy(originalDynamoRule.rule.arn);
```
Comment (markdboyd) [inline packages/api/tests/endpoints/test-rules.js:1384]:
```suggestion
  t.truthy(originalDynamoRule.rule.arn);
  t.truthy(originalEsRecord.rule.arn);
```

File: packages/api/tests/endpoints/test-rules.js [hunk 6]
```diff
@@ -1001,6 +1019,662 @@ test('put() does not write to Dynamo/PostgreSQL if writing to Elasticsearch fail
...
```
Comment (markdboyd) [inline packages/api/tests/endpoints/test-rules.js:1492]:
```suggestion
  t.truthy(originalDynamoRule.rule.arn);
  t.truthy(originalEsRecord.rule.arn);
```

**Review by markdboyd — CHANGES_REQUESTED** [review]
Just a few very minor requested changes to unit tests. Great work!

File: packages/api/tests/endpoints/test-rules.js [hunk 7]
```diff
@@ -1001,6 +1019,662 @@ test('put() does not write to Dynamo/PostgreSQL if writing to Elasticsearch fail
...
```
[the remark this answers is not in the document]
Comment (botanical) [reply packages/api/tests/endpoints/test-rules.js:1611]: 🙇 thank you so much for adding these

**Review by markdboyd — APPROVED** [review]

**Pull request merged**